pkg/option/config.go

```diff
@@ -1570,6 +1576,12 @@ type DaemonConfig struct {
 	// ENITags are the tags that will be added to every ENI created by the AWS ENI IPAM
 	ENITags map[string]string
 
+	// SubnetsIDs are optional subnets IDs used to filter subnets and interfaces listing
+	SubnetsIDs []string
```

Comment:
Please rename to something that indicates that this is being used for IPAM. Same for `SubnetTags`.